common/persistence/sql/sqlplugin/postgres/postgres_server_test.go

```diff
@@ -28,6 +28,38 @@ import (
 	pt "github.com/uber/cadence/common/persistence/persistence-tests"
 )
 
+
+const (
+	testSchemaDir = "schema/postgres"
+)
+
+func getTestClusterOption() *pt.TestBaseOptions {
```

Comment:
There is another one called `GetTestClusterOption` in the package already :D

Reply:
duplicates including testSchemaDir removed ;)